var/spack/repos/builtin/packages/py-sphinxcontrib-bibtex/package.py

```diff
@@ -12,14 +12,20 @@ class PySphinxcontribBibtex(PythonPackage):
     homepage = "https://pypi.python.org/pypi/sphinxcontrib-bibtex"
     url      = "https://pypi.io/packages/source/s/sphinxcontrib-bibtex/sphinxcontrib-bibtex-0.3.5.tar.gz"
 
+    import_modules = ['sphinxcontrib', 'sphinxcontrib.bibtex']
```

Comment:
Spack now automatically detects `import_modules`, this might not be needed anymore